public/app/features/templating/template_srv.ts

```diff
@@ -347,6 +349,55 @@ export class TemplateSrv implements BaseTemplateSrv {
     });
   }
 
+  getVariablesMapInTemplate(
```

Comment:
Or maybe even this to indicate that it is all not only the ones that got a match. I'm not sure which one is best.

```suggestion
  getAllVariablesInTarget(
```